services/thermal_mgr/thermal_mgr.h

```diff
@@ -3,9 +3,14 @@
 #include "lm75bd.h"
 #include "errors.h"
 
+// Temperature Thresholds
+#define TEMP_TH 80
+#define TEMP_HYS 75
+
+
 typedef enum {
   THERMAL_MGR_EVENT_MEASURE_TEMP_CMD,
-  
+  OS_HANDLER_CHECK_OVER_TEMP,
```

Comment:
Add a prefix for `THERMAL_MGR_EVENT` to this event as we want all events of the same type to have the same prefix, makes it easier to understand it